README.md

```diff
@@ -1,13 +1,44 @@
 # Sample Revel Applications
 
-Each directory contains a revel application.
+Each sub directory contains a standalone [Revel](http://revel.github.io/manual)  application, eg
+
+狂欢是好的，保存UTF8 。保持微笑，享受..抱怨..请让它好看;-)
+
 
 ```
 go get github.com/revel/samples
 revel run github.com/revel/samples/booking
 ```
 
-Visit http://revel.github.io for more info
+* [Booking](booking.html) 
+  - A database-driven hotel-booking application, including user management
+  
+* [Chat](chat.html) 
+  - A chat room demonstrating active refresh, long-polling (comet), and [websocket](http://revel.github.io/manual/websockets.html) implementations.
```

Comment:
"Example of active refresh, Comet-style long polling, and WebSocket implementations."